services/configuration/doc/testnet-api-tutorial.md

```diff
@@ -179,7 +179,7 @@ endpoint.
        `validators` field, as specified in `from` field of propose
        transaction. The `from` field is determined by public key of node whose
        `postpropose` endpoint is accessed for signing the transaction on
-       maintainter's behalf.
+       maintainers' behalf.
```

Comment:
I'm not sure that we have to change it to plural form, as well as with the rest of "maintainer's" and "validator's". I don't assert it's wrong, but I really doubt )